core/args.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python3.6
+# -*- coding: utf-8 -*-
+"""
+Functions to parse and retrieve command line arguments.
+"""
+
+import os
+import sys
+import argparse
+
+nbuild_args = None
+nbuild_parser = None
+
+
+def parse_args():
+    """
+    Parses the command line arguments.
+    This function may exit if the command line arguments are invalid.
+    """
+    global nbuild_parser
+    global nbuild_args
+
+    nbuild_parser = argparse.ArgumentParser(
+        description="Compiles packages from a build Manifest."
+    )
+    nbuild_parser.add_argument(
+        '-o',
+        '--output-dir',
+        default=os.path.join(  # Default path is script_dir/packages
+            os.getcwd(),
+            os.path.dirname(sys.argv[0]),
+            'packages',
+        ),
+        help="Output directory for built packages. Default: packages/",
+    )
+    nbuild_parser.add_argument(
+        '-c',
+        '--cache-dir',
+        default=os.path.join(  # Default path is script_dir/cache
+            os.getcwd(),
+            os.path.dirname(sys.argv[0]),
+            'cache',
+        ),
+        help="Cache directory used when downloading and building packages. Default: cache/",
+    )
+    nbuild_parser.add_argument(
+        '--purge',
+        action='store_true',
+        help='Remove all cached data.',
```

Comment:
double quotes :stuck_out_tongue_winking_eye:

Reply:
Damn it :(